spec/Dockerfile_spec.rb

```diff
@@ -24,7 +24,7 @@
                 its(:stdout) { should include_env "GLOBAL_VAR", "global-var" }
                 its(:stdout) { should include_env "PRODUCT_VAR", "product-var" }
                 its(:stdout) { should include_env "SERVICE_VAR", "service-var" }
-                its(:stderr) { should be_empty }
```

Comment:
I'm VERY open to alternative ways of doing this. The match is run against each line of the error, so "random errors" should still pop the test. Also, writing to stderr does add a new line so there will also always be an empty line in addition to the key.

Checking against the whole json message would be fine by me as well, but felt "too brittle" but likely no more brittle than these.

Reply:
I honestly think we forgot about tests for this, so 👍 for adding to the test